src/docs/Foundations/Typography/HeadingStyles/index.mdx

```diff
@@ -20,30 +20,30 @@ Heading styles are patterns meant to help you use consistent typographic hierarc
 - Line-height _(that are optimised for readability and accessibility)_
 - Font-weight _(regular or semi-bold)_
 
-## Weight and line-height variations :
+### Weight and line-height variations :
 
 Each body pattern comes with a modifier to set the font-weight to **'semi bold'** and a modifier to use an alternative **line-height**.
 
-### Heading Small
+### Font-size variations
 
-**Font-size :**  
-Viewport < M (680px) : 23px (1.4375mu)  
-Viewport >= M (680px) : 28px (1.75mu)
+The `font-size` property of headings is a value that varies according to the conditions indicated in the table below:
 
-<Preview path="typographyHeadingSmall" nude />
+| **Name text style / Breakpoints** | **XXL**           | **XL**            | **L**             | **M**           | **S**           |
+| --------------------------------- | ----------------- | ----------------- | ----------------- | --------------- | --------------- |
+| **Large**                         | 3.0625mu (49px)** | 3.0625mu          | 3.0625mu          | 2.5625mu (41px) | 2.125mu (34px)  |
+| **Medium**                        | 2.5625mu (41px)   | 2.5625mu          | 2.5625mu          | 2.125mu (34px)  | 1.75mu (28px)   |
+| **Small**                         | 2.125mu (34px)    | 2.125mu           | 2.125mu           | 1.75mu (28px)   | 1.4375mu (23px) |
 
-### Heading Medium
+Note that by default, headings are set to the **Medium** variation.
 
-**Font-size :**  
-Viewport < M (680px) : 28px (1.75mu)  
-Viewport >= M (680px) : 34px (2.125mu )
+#### Heading Small
 
-<Preview path="typographyHeadingMedium" nude />
+<Preview path="typographyHeadingSmall" nude />
 
-### Heading Large
+#### Heading Medium
+
+<Preview path="typographyHeadingMedium" nude />
 
-**Font-size :**  
-Viewport < M (680px) : 34px (2.125mu)  
-Viewport >= M (680px) : 41px (2.5625mu)
+#### Heading Large
```

Comment:
It could be nice to have some examples of use depending on the size, with simple schematics.

Reply:
@nherwegh , @caqueste this review is for you :)